v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx

```diff
@@ -7,58 +7,47 @@ category: pallet design
 keywords: []
 ---
 
-<Objectives
-  data={[
-    {
-      title: 'Goal',
-      description:
-        "Learn how to use one pallet's capabilities from inside your pallet",
-    },
-    {
-      title: 'Use Cases',
-      description: `Give your pallet access to the methods and types of another pallet.`,
-    },
-    {
-      title: 'Overview',
-      description: `Tight coupling two pallets may suit specific use cases where the outside pallet 
-         contains many types and methods common to those of the initial pallet. If this is not 
-         the case, it's recommended to use loose coupling instead. Read more about pallet 
-         coupling [here](/v3/runtime/pallet-coupling).`,
-    },
-  ]}
-/>
-
-## Steps
-
-### 1. Configure your workspace
-
-Assume you want to use a pallet called `special-pallet`, which is a pallet in your
-local workspace. Import it by providing its path in your pallet's `Cargo.toml` file:
-
-```toml
-special-pallet = { path = '../special-pallet', default-features = false }
-```
-
-### 2. Add a trait bound to your pallet
-
-Now all you need to do is create a trait bound around your pallet's configuration trait:
-
-```rust
-pub trait Config: frame_system::Config + special_pallet::Config {
-    // --snip--
-}
-```
-
-### 3. Use getter function
-
-In order to use a method from `special_pallet`, call it the following way:
-
-```rust
-// Get the members from the vec-set pallet
-let who = special_pallet::Pallet::<T>::get();
-```
-
-The above snippet assumes that `special_pallet` contains a method called `get()`.
+This guide will show you how to use a method from an external pallet inside your pallet.
+
+Tight coupling two pallets may suit specific use cases where an outside pallet contains many types and methods common to those of your initial pallet. 
```

Comment:
I suggest starting with a brief (one sentence-ish) definition of what "tight coupling" means even if it is redundant to information that's somewhere else. Then something like (i'm not sure what you mean by external and outside--does it mean not part of the runtime?):

Tight coupling between pallets can be useful when one pallet requires many of the same types and methods that are defined in another pallet. 
This guide illustrates tight coupling by showing you how to use a method from an external pallet from within your current pallet.

Again I'm not sure about the intention of external, so I'm guessing a little.